controllers/numaresourcesscheduler_controller.go

```diff
@@ -194,3 +200,24 @@ func (r *NUMAResourcesSchedulerReconciler) updateStatus(ctx context.Context, sch
 	}
 	return nil
 }
+
+// a reference which illustrates such technique can be found at https://blog.questionable.services/article/kubernetes-deployments-configmap-change/
+func (r *NUMAResourcesSchedulerReconciler) syncConfigMapHash(ctx context.Context) (string, error) {
```

Comment:
this function is not syncing, it is getting the hash of the current config map. Which is a pre-requisite for syncing, but it is not sending updates to the apiserver. I'd rename like `computeCurrentConfigMapHash`